wallet.go

```diff
@@ -156,19 +174,53 @@ func (w *Wallet) updateNotificationLock() {
 	w.notificationLock = noopLocker{}
 }
 
+// GetCreditAccount returns the first account that can be associated
+// with the given credit.
+// If no account is found, ErrAccountNotFound is returned.
+func (w *Wallet) GetCreditAccount(c txstore.Credit) (uint32, error) {
```

Comment:
Thanks, I'll rename that.

The name is not required in all places and requires a db hit so it might not be worth it to include here.